src/routes/github/github-oauth-router.ts

```diff
@@ -161,6 +165,6 @@ export const GithubAuthMiddleware = async (req: Request, res: Response, next: Ne
 
 // IMPORTANT: We need to keep the login/callback/middleware functions
 // in the same file as they reference each other
-export const GithubOAuthRouter = Router();
+export const GithubOAuthRouter = Router({ mergeParams: true });
```

Comment:
Turns out uuid was an undefined value for any gh paths being set by a nested route. If we want the uuid to be available as a param, we need to pass this property to individual routers, as well as to the subrouter.